java/arcs/android/common/resurrection/DbHelper.kt

```diff
@@ -209,7 +209,7 @@ class DbHelper(
     }
 
     companion object {
-        private const val RESURRECTION_DB_NAME = "resurrection.sqlite3"
+        internal const val RESURRECTION_DB_NAME = "resurrection.sqlite3"
```

Comment:
I think this won't work in g3... Please make sure you get a passing presubmit before submitting!